content/templates/pages/rewrite_links_html.md

```diff
@@ -0,0 +1,5 @@
+---
+hidden: true
+---
+
+{{< template-page "rewrite_links_html" >}}
```

Comment:
Meant to be in a different PR?

Reply:
The docs wouldn't build without this change 🤷‍♂

Reply:
You're right I want ahead and fixed this on https://github.com/cloudflare/workers-docs/pull/624